src/signals.js

```diff
@@ -8,6 +8,7 @@ import { isMobile } from "./helper";
 const defaultSelection = Object.keys(pairs)[0].split("/")[0];
 
 // ui
+export const [buttonLabel, setButtonLabel] = createSignal("");
```

Comment:
Let's not introduce even more global signals. They are only used in `Create` and its children, so let's declare them in `Create` and pass them to the relevant children as props